mono/metadata/gc.c

```diff
@@ -1187,3 +1187,25 @@ mono_gc_is_finalizer_thread (MonoThread *thread)
 {
 	return thread == gc_thread;
 }
+
+void mono_gc_strong_handle_foreach(GFunc func, gpointer user_data)
+{
+	int gcHandleTypeIndex;
+	uint32_t i;
+	const HandleType types[] = { HANDLE_NORMAL, HANDLE_PINNED };
+
+	lock_handles (handles);
+
+	for (gcHandleTypeIndex = 0; gcHandleTypeIndex < 2; gcHandleTypeIndex++)
```

Comment:
Minor, could use `sizeof(types)/sizeof(HandleType)` rather than hard coding the 2.